### PR TITLE
DRAFT: A New Era of Cloud Engineering

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/cloud-engineering-summit/index.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
+---
+title: "A New Era of Cloud Engineering"
+date: 2020-10-01
+meta_desc: "Discover the next evolution for building cloud infrastructure."
+meta_image: cloud_engineering.png
+authors:
+    - sophia-parafina
```
Comment (isaac-pulumi) [inline content/blog/cloud-engineering-summit/index.md:7]: We should add Joe as an author here.

File: content/blog/cloud-engineering-summit/index.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        <img src="charity-majors.jpg" alt="img" style="float:left;display:inline;clear:both;width:100px;height:100px;postion:absolute;border-width:2px;"/>
+        <div style="padding-left:120px;padding-top:5px;padding-bottom:70px;">
+            <b>Testing in Production</b>
+            <ul style="list-style-type:none;">
+                <li>Charity Majors CTO and Co-founder, Honeycomb</li>
+            </ul>
+        </div>
+    </span>
+</div>
+
```
Comment (isaac-pulumi) [inline content/blog/cloud-engineering-summit/index.md:177]: Charity has not responded to email -- we should hold on publishing her name until we know if she is actually going to speak.

File: content/blog/cloud-engineering-summit/index.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+It's time, as an industry, that we come to a consensus of how these complementary cloud Ops integrate and address the challenges of cloud engineering. This effort is much more than playbooks and best practices, cloud engineering attempts to bring all facets of the cloud into an ongoing collaborative framework.
+
+## Cloud Engineering Summit
+
+This month, Pulumi is launching the first-ever Cloud Engineering Summit, in partnership with industry and community experts. Our focus will be on  ways  modern teams are leveraging the cloud to innovate faster than ever before during a period of unprecedented disruption. We’ll also highlight the modern architectures and modern workflows that cloud engineering teams are using to collaborate better, manage complexity and achieve unprecedented scale
+
+The summit brings together cloud engineering leaders and practitioners to share best practices and end-user stories on a broad range of topics with over 23 sessions and 40+ speakers This summit will equip you and your organization to achieve the next level of cloud velocity, innovating faster than ever before, informed and inspired by real-world expertise.
```
Comment (isaac-pulumi) [inline content/blog/cloud-engineering-summit/index.md:58]:
```suggestion
The summit brings together cloud engineering leaders and practitioners to share best practices and end-user stories on a broad range of topics with over 23 sessions and 40+ speakers. This summit will equip you and your organization to achieve the next level of cloud velocity, innovating faster than ever before, informed and inspired by real-world expertise.
```

File: content/blog/cloud-engineering-summit/index.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
+---
+title: "A New Era of Cloud Engineering"
+date: 2020-10-01
```
Comment (isaac-pulumi) [inline content/blog/cloud-engineering-summit/index.md:3]:
```suggestion
date: 2020-10-05
```

**Comment by cnunciato** [comment]
@spara Is this superseded by https://www.pulumi.com/blog/future-of-cloud-engineering/? If so, can we close?

**Pull request closed without merging**